docs/installation/apt_repo.md

```diff
@@ -48,6 +48,10 @@ with the *percona-release* package configuration tool.
 
 6. To make compressed backups, install the `qpress` package:
```

Comment:
This step should be updated to remove the qpress option.

Reply:
Updated